src/components/interior-left-nav/_interior-left-nav.scss

```diff
@@ -248,7 +248,7 @@
 
     .bx--interior-left-nav-collapse {
       cursor: pointer;
-      background-color: rgba(#3d70b2, 0.1);
+      background-color: rgba($link-01, 0.1);
```

Comment:
- [ ] Change to hex and override with `body.bx--v9-theme`  or find a token due to the css custom properties issue or other solution.